modular_citadel/code/modules/arousal/organs/genitals.dm

```diff
@@ -315,6 +335,11 @@
 					S = GLOB.vagina_shapes_list[G.shape]
 				if(/obj/item/organ/genital/breasts)
 					S = GLOB.breasts_shapes_list[G.shape]
+					aroused_state = 0//Breasts don't have aroused_states, there's literally no difference in the spite.
```

Comment:
When I fixed up a little borked up bit of the arousal shitcode, I didn't bother to sprite the arousal state sprites for mammaries and just copypasted the basic ones. So, wait for me to "fix" that, make a different PR for this change, or yell at someone to patch this up, I guess.
Either ways this is out-of-scope.

Reply:
There's about 30 new sprites in this commit though that don't have the aroused state sprites.

Reply:
Then add in a new draw_arousal_state var (defaulting to FALSE) used to prevent genitalia's icon_state from changing when arousal_state is TRUE, and set it TRUE on the bust.
Setting arousal_state on FALSE will break the arousal fluff text on examine. 🤷‍♂